admin/restore.php

```diff
@@ -292,7 +299,7 @@ function restore($filnavn,$backup_encode,$backup_dbtype){
 	if ($popup) {
 		print "<BODY ONLOAD=\"JavaScript:opener.location.reload();\"";
 		print "<meta http-equiv=\"refresh\" content=\"0;URL=../includes/luk.php\">";
```

Comment:
Noticed this during rebasing. I'm sure you are aware, but it is extremely bad coding to put multiple body-tags inside a document. Also meta-tags are only meant to be inside the head-tags.
Sometime, you should clean up this mess across the entire codebase, since browsers may someday stop rendering this correctly without notice.